builder/dispatchers.go

```diff
@@ -261,7 +266,11 @@ func run(b *Builder, args []string, attributes map[string]bool, original string)
 	args = handleJsonArgs(args, attributes)
 
 	if !attributes["json"] {
-		args = append([]string{"/bin/sh", "-c"}, args...)
+		if runtime.GOOS != "windows" {
+			args = append([]string{"/bin/sh", "-c"}, args...)
+		} else {
+			args = append([]string{"cmd", "/S /C "}, args...)
```

Comment:
@duglin - what actually gets executed in docker build (checking on my //build demo machine) is: DEBU[0090] [BUILDER] Command to be executed: [cmd /S /C  powershell -command "&{write-host hello;write-host done}"].

So you're right about the extra space - I'll remove it. It's a hangover from when I was originally double quoting things, and never tidied it up. Thanks for spotting it.